gtsam/linear/GaussianFactorGraph.h

```diff
@@ -57,8 +58,8 @@ namespace gtsam {
     /// The default ordering generation function
     static Ordering DefaultOrderingFunc(
         const FactorGraphType& graph,
-        boost::optional<const VariableIndex&> variableIndex) {
-      return Ordering::Colamd(*variableIndex);
+        const VariableIndex& variableIndex) {
```

Comment:
reference wrapper

Reply:
I was meaning to ask you this. Here it takes an optional but never checks the optional. That's why I replaced it with just a reference. Do you want to keep the optional functionaltiy?

Reply:
Yes.